src/settings_gui.cpp

```diff
@@ -75,7 +75,7 @@ static const void *ResolveVariableAddress(const GameSettings *settings_ptr, cons
  * Get index of the current screen resolution.
  * @return Index of the current screen resolution if it is a known resolution, _resolutions.size() otherwise.
  */
-static uint GetCurRes()
+static uint GetCurrentResolutionIndex()
 {
```

Comment:
`return std::find(_resolutions.begin(), _resolutions.end(), Dimension(_screen.width, _screen.height)) - _resolutions.begin()`
:)
(or std::distance might be slightly more correct)

Reply:
Finger ... hand ... when is the moment I have to rewrite the whole file? :P

Anyway, pick either one; you want the first or the second? :D

Reply:
without `std::distance` it looked horrible. So here you go :)